<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>58</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerName">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>58</y>
     <width>157</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>141</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerScore">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>playerName</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>playerName</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>playerName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>label_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>playerScore</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>playerScore</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>label_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
